client-src/elements/chromedash-all-features-page.ts

```diff
@@ -59,28 +69,28 @@ export class ChromedashAllFeaturesPage extends LitElement {
     }
 
     if (this.rawQuery.hasOwnProperty('q')) {
-      this.query = this.rawQuery['q'];
+      this.query = this.rawQuery['q'] ?? this.query;
     }
     if (this.rawQuery.hasOwnProperty('columns')) {
-      this.columns = this.rawQuery['columns'];
+      this.columns = this.rawQuery['columns'] ?? this.columns;
     }
     if (this.rawQuery.hasOwnProperty('showEnterprise')) {
       this.showEnterprise = true;
     }
     if (this.rawQuery.hasOwnProperty('sort')) {
-      this.sortSpec = this.rawQuery['sort'];
+      this.sortSpec = this.rawQuery['sort'] ?? this.sortSpec;
     }
     if (
       this.rawQuery.hasOwnProperty('start') &&
-      !Number.isNaN(parseInt(this.rawQuery['start']))
+      !Number.isNaN(parseInt(this.rawQuery['start'] as string))
```

Comment:
You shouldn't need the `as string`, once you update the `[key: string]:any` definition to `[key: string]:string`.